cocotb/share/makefiles/Makefile.pylib.Darwin

```diff
@@ -39,6 +39,11 @@ endif
 NATIVE_ARCH=$(shell uname -m)
 ARCH?=$(NATIVE_ARCH)
 
+VERSION?=$(shell $(PYTHON_BIN) -c 'import pkg_resources; print(pkg_resources.get_distribution("cocotb").version)')
```

Comment:
Yes. This is already in but this may break backward compatibility (not `cocotb-config`). This case it will be empty but should not break anything.  Depending on the decision.